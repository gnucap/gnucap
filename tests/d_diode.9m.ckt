simple diode test
V1   1  0  10.
.subckt m4 1 2
	R1   1  2  400.K
	D1   2  0  ddd
.ends
Xm 1 2 m4 $mfactor=4
.model  ddd  d  ( is= 10.f  rs= 0.  n= 1.  tt= 0.  cjo= 1.p  vj= 1.  m= 0.5 
+ eg= 1.11  xti= 3.  kf= 0.  af= 1.  fc= 0.5 )
.list
.print op v(1) v(2) i(V1)
.op
.stat notime
.end
