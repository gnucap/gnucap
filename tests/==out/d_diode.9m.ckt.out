simple diode test 
V1 ( 1 0 )  DC  10.
.subckt m4 ( 1 2 )
R1 ( 1 2 )  400.K
D1 ( 2 0 )  ddd  area=NA( 1.)
.ends m4
Xm ( 1 2 )  m4  $mfactor=4
.model ddd d ( tnom=NA( 27.) is=10.f rs=0. n=1. tt=0. cjo=1.p pb=1. mj=0.5 egap=1.11 xti=3. kf=0. af=1. fc=0.5)
#           v(1)       v(2)       i(V1)     
 27.        10.        0.5582    -94.418u   
Gnucap   System status
iterations: op=9, dc=0, tran=0, fourier=0, total=9
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=2, subckt=0, model=0, total=2
dctran density=100.0%, ac density=100.0%
